.travis.yml

```diff
@@ -96,13 +97,30 @@ jobs:
         - make image/build/ansible
         - make image/push/ansible
 
-    # Build and deploy helm-operator docker image
+    # Build and deploy amd64 helm-operator docker image
     - <<: *deploy
-      name: Docker image for helm-operator
+      name: Docker image for helm-operator (amd64)
+      os: linux
       script:
         - make image/build/helm
         - make image/push/helm
 
+    # Build and deploy ppc64le helm-operator docker image
+    - <<: *deploy
+      name: Docker image for helm-operator (ppc64le)
+      os: linux-ppc64le
+      script:
+        - make image/build/helm
+        - make image/push/helm
+
+    # Build and deploy helm multi-arch manifest list
+      name: Manifest list for helm-operator
+      stage: deploy-manifest
+      services:
+        - docker
+      script:
+        - make image/push/helm-multiarch
```

Comment:
This bit I'm a little out of my wheelhouse on so would love some guidance. I don't *think* we need to do any `before_script` bits, since all you need is docker (covered by the `services` part. I can add the same if `if: type != pull_request AND ( tag IS present OR branch = master OR commit_message =~ /\[travis deploy\]/ ) ` -- or, would you like to create a new commit message check other than `travis deploy`?  
I had initially tested this in our internal Travis env, and it ran at the end after the tests. I'm not sure why it's not running here (as I hadn't added any conditional statements). It definitely has to be run *after* the deploy step completes successfully (so we're guaranteed to have the images pushes for all the architectures). 

I'll push a new commit that's the two adds (the if and moving this to the end of the file), and see what you think else should be done.

Reply:
@joelanford ^

Reply:
The `before_script` bits make sure that all of the branches and tags are fetched from the repo, which I think is required in some cases to get accurate information from the `get_image_tags` function, which is called in `hack/image/push-manifest-list.sh`. And I think we should use the identical `if` statement.

> I had initially tested this in our internal Travis env, and it ran at the end after the tests. I'm not sure why it's not running here (as I hadn't added any conditional statements). It definitely has to be run after the deploy step completes successfully (so we're guaranteed to have the images pushes for all the architectures).

Not sure either. If you have travis set up to watch your fork of operator-sdk, you should see it running there if you include `[travis deploy]` in your commit message.

Reply:
ah, that i wasn't doing.

take a look at what i added to that `.travis.yml` file